draft-ietf-tls-subcerts.md

```diff
@@ -621,22 +553,22 @@ its validity periods, which should also be taken into account.
 
 This document registers the "delegated_credential" extension in the
 "TLS ExtensionType Values" registry.  The "delegated_credential"
-extension has been assigned a code point of 34.  The IANA registry
+extension has been assigned code point 34.  The IANA registry
 lists this extension as "Recommended" (i.e., "Y") and indicates that
 it may appear in the ClientHello (CH), CertificateRequest (CR),
-or Certificate (CT) messages in (D)TLS 1.3 {{RFC8446}} {{!I-D.ietf-tls-dtls13}}.
+or Certificate (CT) messages in (D)TLS 1.3 {{RFC8446}} {{RFC9147}}.
 Additionally, the "DTLS-Only" column is assigned the value "N".
 
 This document also defines an ASN.1 module for the DelegationUsage
 certificate extension in {{module}}.  IANA has registered value 95 for
 "id-mod-delegated-credential-extn" in the "SMI Security for PKIX Module
 Identifier" (1.3.5.1.5.5.7.0) registry.  An OID for the DelegationUsage certificate extension
-is not needed as it is already assigned to the extension from
+is not needed, as it is already assigned to the extension from
 Cloudflare's IANA Private Enterprise Number (PEN) arc.
 
 # Security Considerations
 
-The security consideration documented in this section should be
+The security considerations documented in this section should be
 taken into consideration when using Delegated Certificates.
```

Comment:
{{RFC5280, Section 4.2.1.3}}

Reply:
@bifurcation, please let us know where "{{RFC5280, Section 4.2.1.3}}" should be placed.  Does it replace "this section", or is it additional text?  Also, should "Delegated Certificates" be "Delegated Credentials" here as well?

Reply:
@lbartholomew-rpc this reference does not belong in this section, but I think you are correct Delegated Credentials should replace delegated Certificates in this section and section 5 (Operational Considerations)

Reply:
Delegated Certificates should be Delegated Credentials here. I'm don't think the RFC5280 reference makes sense here as this has been referenced in the appropriate sections. Happy to ignore this comment unless @bifurcation chimes in.

Reply:
I think this comment was added in error.  Please ignore.